server/resources/products/ProductModel.js

```diff
@@ -31,3 +31,16 @@ var productSchema = mongoose.Schema({
 // productSchema.statics.staticFunctionName = function() {};
 
 var Product = mongoose.model('Product', productSchema);
+
+
+// prouct model methods
+function createDefaults() {
+  Product.find({}).exec(function(err, posts) {
```

Comment:
posts => products

doesn't affect the running of it obvi but better to change it